pyproject.toml

```diff
@@ -0,0 +1,142 @@
+readme = "README.md"
+license = {text = "GPL-2.0-or-later"}
+requires-python = ">=3.7"
+authors = [
+    {name = "Michael Rentzsch", email = "michael.rentzsch@repc.de"},
+    {name = "Kai Dietrich", email = "mail@cleeus.de"},
+]
+maintainers = [
+    {name = "Valentin Haenel", email = "valentin@haenel.co"},
+]
+keywords = ["latex", "beamer", "presentation", "wiki"]
+dependencies = []
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v2 or later (GPLv2+)",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Topic :: Text Processing :: Markup",
+    "Topic :: Utilities",
+]
+
+[project.optional-dependencies]
+dev = [
+    "pytest>=7.4.0",
+    "pytest-cov>=6.0.0",
+    "ruff>=0.1.0",
+    "mypy>=1.5.0",
+]
+
+[project.urls]
+Homepage = "https://wiki2beamer.github.io"
+Repository = "https://github.com/wiki2beamer/wiki2beamer"
+Issues = "https://github.com/wiki2beamer/wiki2beamer/issues"
+
+[project.scripts]
+wiki2beamer = "wiki2beamer.cli:cli"
+
+[tool.setuptools]
+package-dir = {"" = "src"}
+
+[tool.setuptools.packages.find]
+where = ["src"]
+
+[tool.ruff]
+target-version = "py37"
+line-length = 100
+fix = true
+show-fixes = true
+output-format = "full"
+
+[tool.ruff.lint]
+select = ["F", "W", "E", "C90", "I", "N", "UP", "ASYNC", "S", "BLE", "FBT", "B", "A", "DTZ", "ISC",
+          "PIE", "T20", "PYI", "PT", "RSE", "RET", "SLF", "SLOT", "SIM", "TCH",
+          "INT", "ARG", "PTH", "TD", "ERA", "PD", "PGH", "PL", "TRY", "FLY", "NPY", "PERF", "FURB", "LOG", "RUF" ]
+ignore = [
+    "A002", # TODO: Fix this
+    "C901", # TODO: Fix this
+    "E203",
+    "E266",
+    "E501",
+    "E731",
+    "E741", # TODO: Fix this
+    "ERA001", # TODO: Fix this
+    "N818", # TODO: Fix this
+    "ISC003", # TODO:Fix this
+    "TD002", # TODO: Fix this
+    "TD003", # TODO: Fix this
+    "PLR0912", # TODO:Fix this
+    "PLR0916", # TODO:Fix this
+    "PLR2004",
+    "PLW2901", # TODO:Fix this
+    "S311"
+]
+
+[tool.ruff.lint.per-file-ignores]
+"tests/*" = [
+    "PLR6301",
+    "PLR0904", # TODO: Fix this
+    "S101",
+    "S603",
+    "S106",
+    "S108",
+]
+
+
+[tool.ruff.format]
+quote-style = "double"
+indent-style = "space"
+line-ending = "lf"
+
+[tool.ruff.lint.isort]
+known-first-party = ["wiki2beamer"]
+
+
+[tool.coverage.report]
+exclude_lines = [
+    "pragma: no cover",
+    "if __name__ == .__main__.:",
+    "pass",
+    "raise NotImplementedError",
+]
+
+[tool.pytest.ini_options]
+testpaths = ["tests"]
+python_files = "test_*.py"
+python_functions = "test_*"
+
+
+[tool.coverage.run]
+source = ["wiki2beamer"]
+omit = ["tests/*"]
```

Comment:
maybe add some spaces below, so as to visually separate the sections!

Reply:
Fixed